### ISSUE TITLE
Error when trying to python manage.py migrate during project setup

### ISSUE THREAD

**Issue by oumsofiane1** [foodoasisla/backend#61]
```
 sofianehome$ python manage.py migrate
Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 316, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python2.7/site-packages/django/conf/__init__.py", line 53, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python2.7/site-packages/django/conf/__init__.py", line 41, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python2.7/site-packages/django/conf/__init__.py", line 97, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/local/Cellar/python/2.7.12_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/sofianehome/code/foodforla/backend/settings.py", line 19, in <module>
    import urllib.parse as urlparse
ImportError: No module named parse
```

**Comment by michaelkunc** [comment]
@oumsofiane1 I'm going to do a clean install of the app in a new environment on my machine and replicate the problem. I *think* I know what's going on here, but I wanna check a few things.

Thanks for opening the issue and the pull request!

**Comment by michaelkunc** [comment]
@oumsofiane1 I was able to recreate the issue. It appears that the virtualenv was created with Python 2.7 rather than 3.5. I'm going to add further instructions so it's clear how to set up the virtualenv.

Thanks again for raising this.

**Comment by oumsofiane1** [comment]
Thank you @michaelkunc

**Comment by michaelkunc** [comment]
Addressed by pull request #65
